docs/_ldi-extensions/ldes-discoverer.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Linked Data Interactions LDES Discoverer
+layout: home
+nav_order: 1
+---
+
+# Linked Data Interactions LDES Discoverer
+
+A lightweight application that discovers the structure of an LDES or a view by retrieving all the tree node relations of
+that LDES or view.
+
+A use case for this could be when you are only interested in a part of the event stream. To know what part you can
+follow, the structure can be discovered first.
+
+## Config
+
+| Property                 | Description                                                         | Required | Default             | Example                   | Supported values                                              |
+|:-------------------------|:--------------------------------------------------------------------|:---------|:--------------------|:--------------------------|:--------------------------------------------------------------|
+| discoverer.url           | Url where from the discoverer needs to start                        | Yes      | N/A                 | http://example.com/my-api | HTTP and HTTPS url                                            |
+| discoverer.source-format | The 'Content-Type' that should be requested to the server.          | No       | application/n-quads | text/turtle               | Any type supported by [Apache Jena](https://jena.apache.org/) |
+| discoverer.output-format | The RDF format that will be used to display all the found relations | No       | text/turtle         | application/ld+json       | Any type supported by [Apache Jena](https://jena.apache.org/) |
```

Comment:
I'm going to leave it as a suggestion, but why not just remove the discoverer prefix?